playbooks/roles/jenkins/defaults/main.yml

```diff
@@ -16,6 +16,30 @@ jenkins_java_args: ""
 jobs: '*'
 jobs_disabled: false
 
+dist_tools:
+  - git
+  - libpq-dev
+  - python-dev
+  - libyaml-dev
+  - libreadline-dev
+  - libffi-dev
+  - postgresql-client-9.5
+  - curl
+  - zip
+  - unzip
+  - ntp
+  - jq
+  - xvfb
+  - wkhtmltopdf
```

Comment:
I think there was an issue at some point where an upgrade to this caused PDFs to be rendered in a much smaller font size. I don't know the exact context (I think Chris handled it) - I just overhead it. Just something to be aware of/mitigate against if the team is planning to bump these libs frequently - there's probably a reasonable risk of regressions somewhere in here. `jq` feels like another lib that _could_ change and break things.

Reply:
Can't find that conversation, but I've done some digging around where these libraries are used:
- `wkhtmltopdf`: the `generate-framework-agreement-counterpart-signature-pages.py` scripts are not run in Docker images, so for now this still needs to be installed in Jenkins
- `jq`: only used in the two scripts where we generate pull requests (`rotate-api-tokens.sh` and `maintenance-mode-pr.sh` ), which are both run in Docker. `jq` is already a dependency in that Dockerfile.

I think the rest we can probably trust ubuntu to update? What do people think?

Reply:
Installing `wkhtmltopdf` in a separate step might be wise, too - like we do with `awscli`.

Reply:
It would be good to understand if the rest of the packages are installed because Jenkins needs them, or because we need them for our jobs... that might be a fair amount of work though.

Reply:
I've done a (not particularly thorough) search and apart from obvious ones like `make`, `git` and `curl`, I can't see direct usage in the jenkins jobs/scripts. However I think George and Chris were pretty thorough when revamping all these ansible playbooks, so I'm sure these are all needed *somewhere* 😸 I guess for now we can pin the ones we know need pinning for certain, let the others upgrade, see if it causes problems? `:this_is_fine_dog:`